routes/cartRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const cartController = require("../controllers/cartController");
+const { loginRequired } = require("../middlewares/auth");
```

Comment:
```suggestion
const express = require("express");

const cartController = require("../controllers/cartController");
const { loginRequired } = require("../middlewares/auth");
```
외부 모듈, 커스텀 모듈간 구분을 위해 줄바꿈을 주면 보다 가독성 좋은 코드가 될 것 같습니다.